_lib/core/blog.php

```diff
@@ -50,6 +50,7 @@ public function blog($options = null)
         $this->table_categories = $options['table_categories'] ?: 'blog_categories';
 
         $this->category_field = array_search($this->table_categories, $opts);
+        $this->limit = $options['limit'] ?: 6;
```

Comment:
I would make this a constant mate. private if its not used outside the class. Public otherwise.

Reply:
do you mean make the default a constant?

Reply:
Exactly (6).